lib/Providers/Time/HttpTimeProvider.php

```diff
@@ -21,10 +21,8 @@ class HttpTimeProvider implements ITimeProvider
     public function __construct(
         public string $url = 'https://google.com',
         public string $expectedtimeformat = 'D, d M Y H:i:s O+',
-        array $options = null,
+        ?array        $options = null,
```

Comment:
I'm afraid this kind of alignment spaces will be removed by an automatic formatter such as phpcs

Reply:
It was PHP-CS-Fixer that put it there in the first place, I've used it as my external formatter. The code style and the code altogether in the PR are entirely validated with php-cs-fixer, phpstan, and unit tests too.

Reply:
very well then! (I was the one adding the .phpcs-fixer config file ^^)